modules/core/02-client/keeper/events.go

```diff
@@ -97,13 +97,15 @@ func emitUpdateClientProposalEvent(ctx sdk.Context, clientID, clientType string)
 	})
 }
 
-// emitUpgradeClientProposalEvent emits an upgrade client proposal event
-func emitUpgradeClientProposalEvent(ctx sdk.Context, title string, height int64) {
+// emitScheduleIBCSoftwareUpgradeEvent emits a schedule IBC software upgrade event
+func emitScheduleIBCSoftwareUpgradeEvent(ctx sdk.Context, title string, height int64, upgradeClientState exported.ClientState) {
 	ctx.EventManager().EmitEvents(sdk.Events{
 		sdk.NewEvent(
-			types.EventTypeUpgradeClientProposal,
+			types.EventTypeScheduleIBCSoftwareUpgrade,
 			sdk.NewAttribute(types.AttributeKeyUpgradePlanTitle, title),
 			sdk.NewAttribute(types.AttributeKeyUpgradePlanHeight, fmt.Sprintf("%d", height)),
+			sdk.NewAttribute(types.AttributeKeyClientType, upgradeClientState.ClientType()),
+			sdk.NewAttribute(types.AttributeKeyConsensusHeight, upgradeClientState.GetLatestHeight().String()),
```

Comment:
Not 100% sure how useful these event additions are. The client type is unlikely to change for the chain and the consensus height should equal the plan height

Reply:
happy to remove if you are